editor/editor_node.cpp

```diff
@@ -923,6 +923,7 @@ void EditorNode::_fs_changed() {
 
 	// FIXME: Move this to a cleaner location, it's hacky to do this in _fs_changed.
 	String export_error;
+	Error err = OK;
```

Comment:
Might be less error prone to add `String export_warning;` and rely on this to print errors and/or warnings and set the exit code accordingly?